treeopt/treeOpt.py

```diff
@@ -10,6 +10,7 @@
 import modules.simulate as simulate
 import modules.optimize as optimize
 import modules.benchmark as benchmark
+#import tests.benchmark as benchmark
```

Comment:
You can remove this import alltogether: The software should not depend on the unit tests or benchmarks used to test the software